apps/docs/components/ArticleHeadingLinks.tsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-useless-escape */
+import { Article, ArticleHeadings } from '@/types/content-types'
+import Link from 'next/link'
+
+export function ArticleHeadingLinks({
+	article,
+	headingLinks,
+}: {
+	article: Article
+	headingLinks: ArticleHeadings
+}) {
+	return headingLinks.length > 1 ? (
```

Comment:
there's a lot of nesting here - can we make this a little less deep?

at the very least, pulling out headingLinks.length ternary check

Reply:
good call!